tests/backup/backup_test.go

```diff
@@ -7737,6 +7571,93 @@ func CreateBackupWithoutCheck(backupName string, clusterName string, bLocation s
 	log.FailOnError(err, "Failed to take backup with request -\n%v", bkpCreateRequest)
 }
 
+// backupSuccessCheck inspects backup task
+func backupSuccessCheck(backupName string, orgID string, retryDuration int, retryInterval int, ctx context.Context) (bool, error) {
```

Comment:
can you move these functions to backup_helper, seems these are common functions used by more tests and later when we split tests to different files it will be useful

Reply:
Moved the functions to backup_helper.go